src/vehicle.h

```diff
@@ -164,6 +164,9 @@ struct vehicle_part : public JsonSerializer, public JsonDeserializer
     /** Try to set fault returning false if specified fault cannot occur with this item */
     bool fault_set( const fault_id &f );
 
+    /** Get wheel diameter times wheel width (inches^2) or return 0 if part is not wheel */
```

Comment:
UTF-8 symbols are supported in source code and practically any font is going to include superscript 2. Readability is king.